outputs.tf

```diff
@@ -65,6 +65,11 @@ output "controllerbuild_iam_role" {
   description = "The IAM Role that the ControllerBuild pod will assume to authenticate"
 }
 
+output "cluster_autoscaler_iam_role" {
```

Comment:
Need to add this to the readme. I will add a pre-commit linter tomorrow to take care of this. I think it gets annoying for contributors.